### PR TITLE
DOCS-2155: Improve iDEAL QR page

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/api/payment_methods/post-order-idealqr.md [hunk 1]
```diff
@@ -110,28 +106,32 @@ Contains:
...
+False: The QR code can only be used once.
 
 `allow_change_amount` | boolean
 
-Set if customers can change the amount to pay. Often used for donations.  
-Required parameters: `max_amount`, or `min_amount`, or both. 
+True: The customer can change the amount to pay.  
+False: The customer cannot change the amount the pay.    
+Required parameters: `max_amount`, or `min_amount`, or both.  
+If you set both `min_amount` and `max_amount` parameters, the `amount` value is ignored.
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-idealqr.md:122]:
```suggestion
```

Comment (Kris-MultiSafepay) [reply content/api/payment_methods/post-order-idealqr.md:122]: I just tested this with the amount set to 2000, min_amount set to 1000, and max_amount set to 3000. 

What happened is that I could select any amount to pay from €10 to €30, but the value defaulted to €20. So it's not ignored, however, I'm not sure whether we should state that the value defaults to the `amount` as it's the expected behaviour IMO.

File: content/api/payment_methods/post-order-idealqr.md [hunk 2]
```diff
@@ -110,28 +106,32 @@ Contains:
...
-If you only use `min_amount`, the `amount` must be more than the `min_amount`. That is, the `amount` = `max_amount`. 
+If `allow_change_amount` is set to `true`, set the `min_amount`.    
+The `min_amount` must not be more than the `amount`, i.e. `amount` = `max_amount`.  
+If `min_amount` is not set, the `amount` value is used as the `min_amount` and vice versa. 
     
 `max_amount` | string
 
-Set the maximum amount if `allow_change_amount option` is set to `true`.  
-If you only use `max_amount`, the `amount` must be less than the `max_amount`.
+If `allow_change_amount option` is set to `true`, set the `max_amount`.
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-idealqr.md:132]:
```suggestion
If `allow_change_amount` is set to `true`, set the `max_amount`.    
```

File: content/api/payment_methods/post-order-idealqr.md [hunk 3]
```diff
@@ -110,28 +106,32 @@ Contains:
...
+If you set both `min_amount` and `max_amount` parameters, the `amount` value is ignored.
 
 `min_amount` | string
 
-Set the minimum amount if `allow_change_amount` is set to `true`.  
-The `min_amount` must not be more than the `amount`.  
-If you only use `min_amount`, the `amount` must be more than the `min_amount`. That is, the `amount` = `max_amount`. 
+If `allow_change_amount` is set to `true`, set the `min_amount`.    
+The `min_amount` must not be more than the `amount`, i.e. `amount` = `max_amount`.  
+If `min_amount` is not set, the `amount` value is used as the `min_amount` and vice versa.
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-idealqr.md:128]: I don't quite understand this. If min_amount isn't set, the amount value is used as min_amount (tested and true). However what does the vice versa imply? If amount isn't set, the min_amount value is used as amount? (tested and doesn't work/fase). Or is this referring to the max_amount parameter having the same behaviour?

File: content/api/payment_methods/post-order-idealqr.md [hunk 4]
```diff
@@ -110,28 +106,32 @@ Contains:
...
+Required parameters: `max_amount`, or `min_amount`, or both.  
+If you set both `min_amount` and `max_amount` parameters, the `amount` value is ignored.
 
 `min_amount` | string
 
-Set the minimum amount if `allow_change_amount` is set to `true`.  
-The `min_amount` must not be more than the `amount`.  
-If you only use `min_amount`, the `amount` must be more than the `min_amount`. That is, the `amount` = `max_amount`. 
+If `allow_change_amount` is set to `true`, set the `min_amount`.    
+The `min_amount` must not be more than the `amount`, i.e. `amount` = `max_amount`.
```
Comment (Kris-MultiSafepay) [inline content/api/payment_methods/post-order-idealqr.md:127]: I don't understand what "i.e. `amount` = `max_amount`" means. If `min_amount` is higher than the `amount` the `amount` will be set to the value of `max_amount`? Is the equals sign ('=') assigning or comparing ('==')?

File: content/payments/methods/banks/ideal/_index.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ weight: 10
 meta_title: "Payment methods - iDEAL - MultiSafepay Docs"
 linktomarketing: "https://www.multisafepay.com/solutions/payment-methods/ideal"
 childlist: '.'
-intro_description: "iDEAL is the leading payment method in the Netherlands and links all major Dutch retail banks. Customers pay via mobile banking app, QR code, or in their own online banking environment. Once a payment is completed, the customer cannot reverse it and iDEAL guarantees settlement."
+intro_description: "iDEAL is the leading payment method in the Netherlands and links all major Dutch retail banks. Customers can pay online via their own banking environment or mobile app, or offline by scanning a QR code on a screen or paper. Customers cannot reverse payment and iDEAL guarantees settlement."
```
Comment (Kris-MultiSafepay) [inline content/payments/methods/banks/ideal/_index.md:7]: How can customers pay offline? The QR code contains a hyperlink that requires an internet connection.

Comment (Kris-MultiSafepay) [reply content/payments/methods/banks/ideal/_index.md:7]: Ah okay, I might get it now. Is that supposed to communicate that the QR code itself doesn't require an active internet connection? E.g. the QR code is printed on a sign on top of a mountain and all that's needed for people to pay is a phone with an active internet connection.

File: content/payments/methods/banks/ideal/product-rules.md [hunk 6]
```diff
@@ -47,8 +47,20 @@ To increase transparency for customers, the name of your website appears on the
...
+
+- iDEAL QR is supported in all our [ready-made integrations](/payments/integrations/). 
+
+- Not all Dutch banks' apps support iDEAL QR yet, so we recommend that customers scan QR codes with their camera or a general QR reader. This redirects to the ideal.nl payment page, which works for all banks. 
+
+- You can specify whether the same QR code can be used more than once.
+
+- You can allow customers to change the amount to pay, e.g. for donations. 
+
+- You can generate more than one payment link for a single order ID. Subsequent transactions each have a unique order ID.
```
Comment (Kris-MultiSafepay) [inline content/payments/methods/banks/ideal/product-rules.md:62]: Screenshot 2021-12-07 at 22 27 37
What is meant by generating more than one payment link for a single order ID? I can scan a QR code multiple times, but if I try to create a QR code (POST /orders) with the same `order_id`, I get the error above.

**Review by Kris-MultiSafepay — CHANGES_REQUESTED** [review]
Major improvements! Couple of questions and a bit of feedback.

File: content/payments/methods/banks/ideal/_index.md (hunk 5 above)
Comment (FrancescaNCook) [reply content/payments/methods/banks/ideal/_index.md:7]:
```suggestion
intro_description: "iDEAL is the leading payment method in the Netherlands and links all major Dutch retail banks. Customers can pay via their own banking environment or mobile app, or by scanning a QR code on a screen or paper. Customers cannot reverse payment and iDEAL guarantees settlement."
```

**Pull request merged**